src/Makefile

```diff
@@ -102,7 +102,7 @@ parser:
 
 # Build the module...
 redisgraph.so: $(MODULE)
-	$(LD) -o $@ $(MODULE) $(SHOBJ_LDFLAGS) -lc -lm
+	$(CC) -o $@ $(MODULE) $(SHOBJ_LDFLAGS) -lc -lm
```

Comment:
@swilly22 For GCC, this seems like the right change - `ld` can't resolve `-lgomp` because the OpenMP interface is held in GCC internals. To use `ld`, we'd have to locate `libgomp.a` or `libgomp.so` on disk, then add the path as a `-L` argument. This seems a lot more straightforward, but I don't know if there are consequences I'm not considering (or if it will even work with Clang, though if it doesn't, we can resolve that).